src/services/imageService.ts

```diff
@@ -1,15 +1,16 @@
 import axios from "axios";
 import serviceSuccessInterceptor from "./serviceSuccessInterceptor";
 
-export const IMAGE_SERVER_BASE_URL = "https://images.dataprick.com";
+export const IMAGE_SERVER_BASE_URL = "http://localhost:3010";
```

Comment:
LOL, not going live with this hopefully

Reply:
env var?

Reply:
Chill dude, it's for demo, so you can test. No need for env, we will change it before production.

Reply:
Ok cool then